docs/examples/.env.truenas.example

```
@@ -0,0 +1,64 @@
+# ============================================
+# DATABASE
+# ============================================
+POSTGRES_USER=postgres
+POSTGRES_PASSWORD=postgres
+POSTGRES_DB=fallout_db
+
+# ============================================
+# INITIAL ADMIN USER
+# ============================================
+FIRST_SUPERUSER_USERNAME=ElderEvil
+FIRST_SUPERUSER_EMAIL=elder.evil.dev@proton.me
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The FIRST_SUPERUSER_EMAIL key should go before the FIRST_SUPERUSER_USERNAME key

(UnorderedKey)